src/webui/www/private/confirmtrackerdeletion.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Remove tracker)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEvent("domready", () => {
+            const url = new URI().getData("url");
+            $("confirmDeleteTrackerText").innerHTML = 'QBT_TR(Are you sure you want to remove tracker "%1" from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", decodeURIComponent(url));
```

Comment:
You should refrain `innerHTML` whenever possible. It is dangerous and slow performance-wise.
```suggestion
            $("confirmDeleteTrackerText").textContent = 'QBT_TR(Are you sure you want to remove tracker "%1" from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", decodeURIComponent(url));
```